examples/text_labels_and_annotations/shaded_table.py

```diff
@@ -0,0 +1,73 @@
+from matplotlib import pyplot as plt
```

Comment:
ok -- so why does the contour example you pointed me at >explicitly< save the figure?